simapp/go.mod

```diff
@@ -1,6 +1,8 @@
 module cosmossdk.io/simapp
 
-go 1.21
+go 1.22
+
+toolchain go1.22.0
```

Comment:
Any reason we have a `toolchain` directive here ?

Reply:
It was autogenerated when uploading the dependencies via the `go mod tiny` script